definitions/sources/chrome-ux-report.js

```diff
@@ -0,0 +1,40 @@
+const database = 'chrome-ux-report'
+
+const pastMonth = constants.fnPastMonth(constants.currentMonth)
+const pastMonthYYYYMM = pastMonth.substring(0, 7).replace('-', '')
+
+declare({
+  database,
+  schema: 'materialized',
+  name: 'country_summary'
+})
+
+assert('country_summary_not_empty').query(ctx => `
+SELECT
+  'No data for the specified date' AS error_message
+FROM ${ctx.ref(database, 'materialized', 'country_summary')}
+WHERE yyyymm = ${pastMonthYYYYMM}
+GROUP BY yyyymm
+HAVING COUNT(0) < 20000000
```

Comment:
Any explanation for this magic number?

Reply:
I tried to get from 0 rows to a better completeness estimation.
But I agree that was a magic number. Adjusted it to check for dimensions uniqueness (in case it's updated sequentially, partition after partition).